### PR TITLE
Several agreed small code reorganizations

### PR DESCRIPTION
**Pull request by ancorgs**

### CODE REVIEW

**Comment by ancorgs** [comment]
The github view is doing quite interesting wrong assumptions like
`lib/storage/refined_devicegraph.rb → /examples/proposal_dump_yaml.rb`

Obviously, that's not a moved file. They are completely different files committed in different commits. So please review in a per-commit basis to avoid those artifacts.

File: src/lib/storage/planned_volumes_list.rb [hunk 1]
```diff
@@ -29,7 +29,7 @@ module Storage
     # Set of PlannedVolume elements
     #
     # Implements Enumerable and provides some extra methods to query the set
```
Comment (aschnell) [inline src/lib/storage/planned_volumes_list.rb]: The set of what? And why set if the class is called List?

**Comment by aschnell** [comment]
Concerning refining the Disk class. Do you know of the downcast pitfall (see https://github.com/openSUSE/libstorage-ng/blob/master/bindings/storage-downcast.i)?

**Comment by aschnell** [comment]
Otherwise LGTM.

File: src/lib/storage/planned_volumes_list.rb (hunk 1 above)
Comment (ancorgs) [inline src/lib/storage/planned_volumes_list.rb]: It comes from the previous line "Set of PlannedVolume elements". But I can clarify if needed. Using "set" just as a synonym for "list" or "collection" with no particular reason. I will rephrase it a bit.

**Pull request merged**